sdks/python/apache_beam/examples/inference/README.md

```diff
@@ -0,0 +1,111 @@
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# Example RunInference API Pipelines
+
+This module contains example pipelines that use the Beam RunInference
+API. <!---TODO: Add link to full documentation on Beam website when it's published.-->
+
+## Pre-requisites
+
+You must have `apache-beam>=2.40.0` installed in order to run these pipelines,
+because the `apache_beam.examples.inference` module was added in that release.
+```
+pip install apache-beam==2.40.0
+```
+
+### Pytorch dependencies
+The RunInference API has support for the Pytorch framework. To use Pytorch locally, first install `torch`.
+```
+pip install torch==1.11.0
+```
+
+For installation of the `torch` dependency for Dataflow pipelines, refer to these
+[instructions](https://beam.apache.org/documentation/sdks/python-pipeline-dependencies/#pypi-dependencies).
+
+<!---
+TODO: Add link to full documentation on Beam website when it's published.
+
+i.e. "See the
+[documentation](https://beam.apache.org/documentation/dsls/dataframes/overview/#pre-requisites)
+for details."
+-->
+
+### Datasets and Models for RunInference
+Data related to RunInference has been staged in
+`gs://apache-beam-ml/` for use with these example pipelines. You can see this by using the [gsutil tool](https://cloud.google.com/storage/docs/gsutil#gettingstarted).
+```
+gsutil ls gs://apache-beam-ml
+```
+
+---
+## Image Classification with ImageNet dataset
+
+[`pytorch_image_classification.py`](./pytorch_image_classification.py) contains
+an implementation for a RunInference pipeline thatpeforms image classification
+on [ImageNet dataset](https://www.image-net.org/) using the MobileNetV2
+architecture.
+
+The pipeline reads the images, performs basic preprocessing, passes them to the
+PyTorch implementation of RunInference, and then writes the predictions
+to a text file in GCS.
+
+### Dataset and model for Image Classification
+
+<!---
+TODO: Add once benchmark test is released
+- `gs://apache-beam-ml/testing/inputs/imagenet_validation_inputs.txt`:
+  text file containing the GCS paths of the images of all 5000 imagenet validation data
+    - gs://apache-beam-ml/datasets/imagenet/raw-data/validation/ILSVRC2012_val_00000001.JPEG
+    - ...
+    - gs://apache-beam-ml/datasets/imagenet/raw-data/validation/ILSVRC2012_val_00050000.JPEG
+-->
+- `gs://apache-beam-ml/testing/inputs/it_imagenet_validation_inputs.txt/`:
+  text file containing the GCS paths of the images of a subset of 15 imagenet
+  validation data
+    - gs://apache-beam-ml/datasets/imagenet/raw-data/validation/ILSVRC2012_val_00000001.JPEG
+    - ...
+    - gs://apache-beam-ml/datasets/imagenet/raw-data/validation/ILSVRC2012_val_00000015.JPEG
```

Comment:
(optional) It might be nice to clarify that these sub-bullets are the file contents with something like:
```sh
$ gsutil cat gs://apache-beam-ml/testing/inputs/it_imagenet_validation_inputs.txt
gs://apache-beam-ml/datasets/imagenet/raw-data/validation/ILSVRC2012_val_00000001.JPEG
...
gs://apache-beam-ml/datasets/imagenet/raw-data/validation/ILSVRC2012_val_00000015.JPEG
```

Reply:
Added.